server/utils/socketConfig.py

```diff
@@ -0,0 +1,15 @@
+from extensions import app, socketio
+from flask_socketio import send, emit, join_room, leave_room, SocketIO
+
+@socketio.on('connect')
+def connected():
+    emit('message', "You have connected")
+
+@socketio.on('subscribeToNotifications')
+def joined(user_id):
+    join_room(user_id)
```

Comment:
It would be really nice to authenticate the user here once they subscribe to notifications, by passing their localstorage token in one of the early connections - since someone could listen to these notifications if they just connect to the right room

Reply:
Added in TODO. Created a task